kernel/src/main/scala/org/typelevel/sbt/TypelevelKernelPlugin.scala

```diff
@@ -37,6 +38,36 @@ object TypelevelKernelPlugin extends AutoPlugin {
           BasicCommands.addAlias(BasicCommands.removeAlias(s, name), name, contents)
         }
       })
+
+    private[sbt] lazy val currentReleaseImpl = Def.setting {
```

Comment:
Alternatively we can remove this nasty `Impl` suffix, at the cost of having to avoid full imports at the use site in the versioning plugin